File: Assignment2_PF/Python_Programs/P18_Calculate_S_Using_Formula.py
# Output Preview Link:
# https://www.programiz.com/online-compiler/9etsgDIgz84Hh


vi = float(input("Enter initial velocity (vi): "))
t = float(input("Enter time (t) in seconds: "))
a = float(input("Enter acceleration: "))

s = vi * t + 0.5 * a * t**2
print("Displacement (s) =", s)

File: Assignment2_PF/C++_Programs/P18_Calculate_S_Using_Formula.cpp
// Output Preview Link:
// https://onlinegdb.com/_kNhZ9A8h

#include <iostream>
using namespace std;

int main() {
    float vi, t, a, s;

    cout << "Enter initial velocity (vi): ";
    cin >> vi;

    cout << "Enter time (t): ";
    cin >> t;

    cout << "Enter acceleration (a): ";
    cin >> a;

    s = vi * t + 0.5 * a * t * t; 

    cout << "S = " << s << endl;
    return 0;
}
